CHANGELOG.md

```diff
@@ -47,6 +47,8 @@
   deprecated.
   [#1792](https://github.com/Kong/kong-operator/pull/1972)
   <!-- https://github.com/Kong/kong-operator/issues/1944 -->
+- Removed `KongIngress`, `TCPIngress` and `UDPIngress` CRDs together with their controllers.
```

Comment:
I believe we should link here to docs explaining the migration for users using these.

Reply:
Added https://github.com/Kong/kong-operator/pull/1971/commits/00e9547cfe53dfd4594337d37eb1134cdcd3fa10

Reply:
Nice ! 👍